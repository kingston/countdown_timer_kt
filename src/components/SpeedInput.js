import React from 'react';

function SpeedInput() {
  return <div />;
}

export default SpeedInput;
