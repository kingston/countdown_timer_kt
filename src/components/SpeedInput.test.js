import React from 'react';
import SpeedInput from './SpeedInput';
import { shallow } from 'enzyme';

it('renders without crashing', () => {
  shallow(<SpeedInput />);
});
