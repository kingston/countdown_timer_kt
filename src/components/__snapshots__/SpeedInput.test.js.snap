// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`activates button based on speed 1`] = `
<ButtonGroup>
  <Button
    active={false}
    as="button"
    key="1"
    onClick={[Function]}
  >
    1.0
    X
  </Button>
  <Button
    active={false}
    as="button"
    key="1.5"
    onClick={[Function]}
  >
    1.5
    X
  </Button>
  <Button
    active={true}
    as="button"
    key="2"
    onClick={[Function]}
  >
    2.0
    X
  </Button>
</ButtonGroup>
`;
